system-tests/fix/fix.test.js

```diff
@@ -134,3 +134,34 @@ describe("fix", () => {
       });
   });
 });
+
+describe("fix with BOM", () => {
+  let tmpDir;
+  let stylesheetPath;
+
+  beforeEach(() => {
+    tmpDir = os.tmpdir();
+    stylesheetPath = path.join(tmpDir, `stylesheet-with-bom.css`);
+
+    return cpFile(path.join(__dirname, "stylesheet.css"), stylesheetPath);
+  });
+
+  afterEach(() => {
+    return del(stylesheetPath, { force: true });
+  });
+
+  // eslint-disable-next-line jest/no-disabled-tests
+  it.skip("doesn't strip BOM", () => {
+    return stylelint
+      .lint({
+        files: [stylesheetPath],
+        configFile: systemTestUtils.caseConfig("fix"),
+        fix: true
+      })
+      .then(() => {
+        return pify(fs.readFile)(stylesheetPath, "utf8").then(fileContent => {
```

Comment:
I need to replace `pify` here https://github.com/stylelint/stylelint/commit/14a0da663a52da10ca8df9a0183294e64e4f66fe.

Reply:
Fixed.